cif_ed.dic

```diff
@@ -856,4 +856,8 @@ save_
        core CIF or other relevant COMCIFS-managed CIF dictionaries. Some
        of the definitions in this dictionary may be copies of core
        definitions with extended attributes.
+;
+         1.0.1-dev                2025-02-19
+;
+       Initialised new development version.
```

Comment:
```suggestion
       # Please update the date above and describe the change below until
       # ready for the next release

       Initialised new development version.
```

Reply:
Just a standard boilerplate pseudocomment to make it more clear what is expected.